bitnami/discourse/README.md

```diff
@@ -50,178 +50,179 @@ The command removes all the Kubernetes components associated with the chart and
 
```

Comment:
Updated this file using vscode's https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one 🤔